### PR TITLE
Add Angular 7 scaffold and improve linting

### PR DESCRIPTION
**Pull request by ptesser**

### CODE REVIEW

File: front-end/src/app/app.module.ts [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import { NgModule } from '@angular/core';
+import { BrowserModule } from '@angular/platform-browser';
+
```
Comment (Vannnu) [inline front-end/src/app/app.module.ts:3]: why empty Line?

File: front-end/src/main.ts [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import { enableProdMode } from '@angular/core';
+import { platformBrowserDynamic } from '@angular/platform-browser-dynamic';
+
```
Comment (Vannnu) [inline front-end/src/main.ts:3]: The same

**Review by Vannnu — CHANGES_REQUESTED** [review]

File: front-end/src/app/app.module.ts (hunk 1 above)
Comment (ptesser) [reply front-end/src/app/app.module.ts:3]: Because separate external libs from internals one.

File: front-end/src/main.ts (hunk 2 above)
Comment (ptesser) [reply front-end/src/main.ts:3]: Because it separates external libs from internals one.

**Pull request merged**